### DISCUSSION TITLE
先天打灰圣体

### DISCUSSION THREAD

**Discussion by fzdqz** [share121/inter-knot#4834] [绝区零]
![Screenshot_2024-09-13-17-51-13-488_com tencent mm](https://github.com/user-attachments/assets/89fa9f29-ba1e-4e11-bc49-204145df3ea0)
这就是大酋长〒_〒

**Comment by Hangover-Cyan-Eternally** [comment]
我鲨鱼妹也是吃满大保底😭

**Comment by 070612wwz** [comment]
和我一样
![Uploading img-143565083_1724839194076.jpg…]()

**Comment by huduguo** [comment]
那我的小保底是歪到你们身上了，一次没歪

**Comment by sb35386057** [comment]
![Uploading Screenshot_2024-09-16-08-59-11-996_com.tencent.mm.jpg…]()

**Reply by sb35386057** [reply]
吸吧孩子

**Reply by lllliufeng** [reply]
你这没传上来罢（恼）<img src="https://bbs-static.miyoushe.com/static/2023/05/22/a2fa805a0a782cdb8ab44ddaf8d5343d_9213146029750742963.png" alt="阿姬倒地了02" width="75" >

**Comment by sb35386057** [comment]
![Screenshot_2024-09-16-08-59-11-996_com tencent mm](https://github.com/user-attachments/assets/fe2af105-4410-4383-8388-e6926ca89395)
